recipes/maturin/license-mit

```diff
@@ -0,0 +1,25 @@
+Copyright (c) 2018 konstin
```

Comment:
Can drop this file. It appears to already be in the source we are downloading.

Reply:
Done